templates/tools/dockerfile/test/android_ndk/Dockerfile.template

```diff
@@ -14,44 +14,34 @@
   # See the License for the specific language governing permissions and
   # limitations under the License.
   
-  FROM debian:10
+  FROM debian:11
   
   <%include file="../../apt_get_basic.include"/>
   <%include file="../../run_tests_python_deps.include"/>
   <%include file="../../cmake.include"/>
   <%include file="../../run_tests_addons.include"/>
   RUN apt update && apt upgrade -y
 
-  # Java required by Android SDK (using Eclipse Temurin Package)
-  RUN apt install -y wget apt-transport-https && ${'\\'}
-      mkdir -p /etc/apt/keyrings && ${'\\'}
-      wget -O - https://packages.adoptium.net/artifactory/api/gpg/key/public | tee /etc/apt/keyrings/adoptium.asc && ${'\\'}
-      echo "deb [signed-by=/etc/apt/keyrings/adoptium.asc] https://packages.adoptium.net/artifactory/deb $(awk -F= '/^VERSION_CODENAME/{print$2}' /etc/os-release) main" | tee /etc/apt/sources.list.d/adoptium.list
-  RUN apt update && apt install -y temurin-8-jdk
-
-  RUN apt-get -y autoremove && ${'\\'}
-      rm -rf /var/lib/apt/lists/*
+  # Java
+  RUN apt-get install -y openjdk-11-jdk
 
   # Install Android SDK
-  ENV ANDROID_SDK_VERSION 4333796
-  RUN mkdir -p /opt/android-sdk && cd /opt/android-sdk && ${'\\'}
-      wget -q https://dl.google.com/android/repository/sdk-tools-linux-$ANDROID_SDK_VERSION.zip && ${'\\'}
-      unzip -q sdk-tools-linux-$ANDROID_SDK_VERSION.zip && ${'\\'}
-      rm sdk-tools-linux-$ANDROID_SDK_VERSION.zip
-  ENV ANDROID_SDK_PATH /opt/android-sdk
   ENV ANDROID_HOME /opt/android-sdk
-  RUN yes | $ANDROID_SDK_PATH/tools/bin/sdkmanager --licenses  # accept all licenses
-
-  # Install Android NDK & CMake
-  # This is not required but desirable to reduce the time to download and the chance of download failure.
-  RUN mkdir -p ~/.android && touch ~/.android/repositories.cfg
-  RUN $ANDROID_SDK_PATH/tools/bin/sdkmanager 'ndk;25.1.8937393' 'cmake;3.22.1'
+  RUN mkdir -p $ANDROID_HOME && ${'\\'}
+      wget -O cmd.zip https://dl.google.com/android/repository/commandlinetools-linux-9477386_latest.zip && ${'\\'}
```

Comment:
I didn't find a better link, but it seems we may encounter issues down the road when `latest` has different contents.

Reply:
(This is maybe not actionable)

Reply:
`9477386` part is the version so I'm not sure what latest part actually means here.